crates/sncast/tests/e2e/account/create.rs

```diff
@@ -337,33 +329,52 @@ pub async fn test_keystore_without_account() {
         DEVNET_OZ_CLASS_HASH,
     ];
 
-    let snapbox = runner(&args);
+    let snapbox = runner(&args).current_dir(temp_dir.path());
 
     snapbox.assert().stderr_matches(indoc! {r"
         command: account create
         error: Argument `--account` must be passed and be a path when using `--keystore`
     "});
-
-    _ = fs::remove_file(keystore_path);
 }
 
-#[test_case("tests/data/keystore/my_key.json", "tests/data/keystore/my_account_new.json", "error: Keystore file tests/data/keystore/my_key.json already exists" ; "when keystore exists")]
-#[test_case("tests/data/keystore/my_key_new.json", "tests/data/keystore/my_account.json", "error: Account file tests/data/keystore/my_account.json already exists" ; "when account exists")]
+#[test_case("./tests/data/keystore/my_key.json", "./tests/data/keystore/my_account_new.json", "error: Keystore file my_key.json already exists" ; "when keystore exists")]
+#[test_case("./tests/data/keystore/my_key_new.json", "./tests/data/keystore/my_account.json", "error: Account file my_account.json already exists" ; "when account exists")]
 pub fn test_keystore_already_exists(keystore_path: &str, account_path: &str, error: &str) {
+    let temp_dir = tempdir().expect("Unable to create a temporary directory");
+
+    let keystore_path = Path::new(keystore_path);
+    if keystore_path.exists() {
+        fs_extra::file::copy(
+            keystore_path,
+            temp_dir.path().join(keystore_path.file_name().unwrap()),
+            &fs_extra::file::CopyOptions::new().overwrite(true),
+        )
+        .expect("Unable to copy keystore file");
+    }
+    let account_path = Path::new(account_path);
+    if account_path.exists() {
+        fs_extra::file::copy(
+            account_path,
+            temp_dir.path().join(account_path.file_name().unwrap()),
+            &fs_extra::file::CopyOptions::new().overwrite(true),
+        )
+        .expect("Unable to copy account file");
+    }
+
     let args = vec![
         "--url",
         URL,
         "--keystore",
-        keystore_path,
+        keystore_path.file_name().unwrap().to_str().unwrap(),
```

Comment:
we could just construct a str representation of the path straight away, since we do not need Path representation, right? this could simplify such lines, I guess

Reply:
We use a Path representation for `.exists()` method

Reply:
we can use absolute path though, right? it should not matter for the test case, until the file exists